modules/swagger-codegen/src/main/resources/go-server/main.mustache

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	sw "./{{apiPath}}"
```

Comment:
Hm, I don't think you understand :) Relative import paths compile _in some circumstances_ to aid in very specific testing scenarios. But "Go programs cannot use relative import paths within a work space" so this breaks as soon as the user puts this code into their project. In fact, the "feature" is so often mis-used, that it is being discussed to remove it from the language entirely. See https://golang.org/cmd/go/#hdr-Relative_import_paths. 

I don't know much about how swagger-codegen works. But, I would expect to run some CLI tool from my project directory, which would be in my $GOPATH. And I'd expect to provide the fully-qualified import path prefix. So something like this

```
$ cd $GOPATH/src/github.com/peterbourgon/myproject
$ swagger-codegen --import-prefix=github.com/peterbourgon/myproject --output-dir=myapi resources/swagger.spec
reading resources/swagger.spec
putting generated code in myapi/
using import path "github.com/peterbourgon/myproject/myapi" for generated code
writing main.go
writing logger.go
writing controller.go
done
$
```

`--import-prefix` could be deduced from the current context (reading GOPATH) and `--output-dir` could have some sensible default.

If this isn't possible, the next-best thing would be to put a big WARNING or DISCLAIMER around that import statement.

```go
import (
    // WARNING!
    // Change this to a fully-qualified import path
    // once you place this file into your project.
    // For example,
    //    sw "github.com/myname/myrepo/{{apiPath}}"
    sw "./{{apiPath}}"
)
```